packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -440,7 +440,7 @@ export const installDependencies = async () => {
 };
 
 // Note that Node 20 <20.2.0 leads to build errors: `Unexpected early exit.`
-const NODE_ENGINES = "^18.0.0 || >=20.2.0";
+const NODE_ENGINES = "^18.20.0 || >=20.2.0";
```

Comment:
As I mentioned in Slack, I don't think this is the correct fix. This wasn't a problem until the dep upgrade PR and searching the error has results that it's related to types/node.

Reply:
I tried reverting the @types/node version as used by pages back to ^20.10.4 for the branch/commit here:
https://www.yext.com/s/950642/yextsites/138238/branch/102492/deploys/recent

Reverting @types/node version did not resolve the issue, but changing the node version to 18.20 does resolve the issue. I also confirmed that 18.19 also works.

Reply:
Wrong version. You put 18.40.0

Reply:
Also update the title